udata_front/views/base.py

```diff
@@ -92,9 +93,12 @@ class SearchView(Templated, BaseView):
     model = None
     context_name = 'objects'
     search_adapter = None
+    page_size: Optional[int] = None
 
     def get_queryset(self):
         parser = self.search_adapter.as_request_parser()
+        if self.page_size:
+            parser.replace_argument('page_size', type=int, location='args', default=self.page_size)
```

Comment:
@quaxsze, do you have any better suggestion on how to override default page_size on ModelSearchAdapter parser ?

Reply:
After discussion, seems that we can improve the readability by making `page_size` configurable in the adapter in udata.
I created [this ticket](https://github.com/etalab/data.gouv.fr/issues/880), so that we can merge this PR and update in a second iteration.